src/lib.rs

```diff
@@ -84,6 +84,18 @@
 //! let source = source.take_duration(Duration::from_secs(5)).repeat_infinite();
 //! ```
 //!
+//! ## Alternative Decoder Backends
+//!
+//! [Symphonia](https://github.com/pdeljanov/Symphonia) is an alternative deocder library that can be used in place of many of the default backends.
+//! Currently, the main benefit is that Symphonia is the only decoder library that supports M4A and AAC,
+//! but it may be used to implement additional optional functionality in the future.
+//!
+//! To enable, set `default-features = false` and enable either the `symphonia-all` feature to enable all Symphonia codecs
```

Comment:
This makes it sound like `default-features` has to be set to false which is not the case. Maybe change the wording to make it more sound like a reccomendation?